style-bibliographique-des-collections-des-shp-ephe-paris.csl

```diff
@@ -466,8 +466,10 @@
       </else-if>
     </choose> 
   </macro>
-  <macro name="isbn">
+  <macro name="ISBN">
+   <group>  
     <text variable="ISBN" prefix=". ISBN : " suffix="."/>
+   </group> 
   </macro>
```

Comment:
You don't use this macro here anywhere in your code and therefore you should completely delete it completeley, i.e. delete the 5 lines:
```
   <macro name="ISBN">
   <group>  
     <text variable="ISBN" prefix=". ISBN : " suffix="."/>
   </group> 
   </macro>
```